### PR TITLE
New acceptAllFriendRequests

### PR DESCRIPTION
**Pull request by eremeyko**
The old script gives the following error due to the limitation of the number of requests (25):
`{"error":{"error_code":13,"error_msg":"Runtime error occurred during code invocation: Too many API calls","request_params":[{"key":"user_id","value":"690564895"}]}}`

You can accept all applications by constant requests until the first error

### CODE REVIEW

File: Application/acceptAllFriendRequests.js [hunk 1]
```diff
@@ -2,7 +2,10 @@ var requests = API.friends.getRequests().items;
 var i = 0;
 while(i < requests.length)
 {
-  API.friends.add({ user_id: requests[i] });
+  if(i < 24)
+  {
+    API.friends.add({ user_id: requests[i] });
+  }
```
Comment (konard) [inline Application/acceptAllFriendRequests.js]:
```suggestion
  API.friends.add({ user_id: requests[i] });
```

File: Application/acceptAllFriendRequests.js [hunk 2]
```diff
@@ -2,7 +2,10 @@ var requests = API.friends.getRequests().items;
 var i = 0;
```
Comment (konard) [inline Application/acceptAllFriendRequests.js:2]:
```suggestion
var requests = API.friends.getRequests({ count: 23 }).items;
var i = 0;
```

**Review by konard — COMMENTED** [review]
Maybe it is better to limit the amount requested friend requests?

**Comment by eremeyko** [comment]
Then count is up to 24, because one request is to get a list of incoming requests

**Review by konard — APPROVED** [review]

**Pull request merged**